modules/enterData.js

```diff
@@ -0,0 +1,43 @@
+const urlPOST = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/mwHLZWvmcAi40ernHtsJ/scores/';
+const name = document.getElementById('inpIdName');
+const score = document.getElementById('inpIdScore');
+const submit = document.getElementById('submitBtn');
+
+let gid = [];
+
+const enterData = () => {
+  name.value = '';
+  score.value = '';
+
+  submit.addEventListener('click', () => {
+    if (name.value && score.value) {
+      fetch(urlPOST, {
+        method: 'POST',
+        body: JSON.stringify({
+          user: name.value,
+          score: score.value,
+        }),
+        headers: {
+          'Content-type': 'application/json; charset=UTF-8',
+        },
+      })
+        .then((response) => {
+          if (!response.ok) {
+            throw new Error('From POST: Network response was not ok');
+          }
+          return response.json();
+        })
+        .then((data) => {
+          gid = data.result;
+          console.log(gid);
+          name.value = '';
+          score.value = '';
+        })
+        .catch((error) => {
+          console.error('Error:', error);
+        });
+    }
```

Comment:
- [ ] The project requirement as shown below requires you to make use of JavaScript's `async & await` to make API calls as opposed to using the `.then` like you have done. I would kindly suggest you make the necessary adjustment to your functions by using `async & await`
- * ## Project requirement
![prr](https://github.com/Nelofarzabi/Leaderboard/assets/89055502/57aaa6e0-cfc8-4cf7-9c4f-4a78600c77ff)